examples/statemachine/statemachine.py

```diff
@@ -8,14 +8,7 @@
 import os
 import types
 import importlib
-try:
-    import urllib.parse
-    url_parse = urllib.parse.urlparse
-except ImportError:
-    print("import error, Python 2 not supported")
-    raise
-    import urllib
-    url_parse = urllib.parse
+from urllib.parse import urlparse
```

Comment:
importing from a py2/py3 module would be better.

I agree that the try/except is a mess